components/molecule/rating/src/index.scss

```diff
@@ -0,0 +1,84 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+@import '~react-rating-svg/lib/scss/react-rating-svg';
+
+$fz-molecule-rating-small: $fz-s !default;
+$fz-molecule-rating-medium: $fz-m !default;
+$fz-molecule-rating-large: $fz-l !default;
+
+$c-molecule-rating: #5f5f5f !default;
```

Comment:
colors should be one of our tokens, not hardcoded

Reply:
fixed!